drivers/gpu/drm/vc4/vc4_kms.c

```diff
@@ -436,14 +431,6 @@ static void vc4_atomic_commit_tail(struct drm_atomic_state *state)
 		drm_dbg(dev, "Running the core clock at %lu Hz\n",
 			new_hvs_state->core_clock_rate);
 
-		/*
-		 * Request a clock rate based on the current HVS
-		 * requirements.
-		 */
-		hvs->core_req = clk_request_start(hvs->core_clk,
```

Comment:
Should this be replaced by a call to `clk_set_min_rate`?

Reply:
It was added in the next patch but you're right that it makes more sense to have it there. I've reworked the branch a bit and pushed a new version.